### ISSUE TITLE
Setting the height of custom bar view results in inconsistent UI of UITabBar is height is more than 100

### ISSUE THREAD

**Issue by atulvuliv** [LeoNatan/LNPopupController#180]
First of all thank you for your amazing work. We are using this library in our existing app and it looks perfect.

There is however one issue that we are facing right now in our latest feature when we use the Custom Bar View Controller by subclassing `LNPopupCustomBarViewController`. (We have a `UITabBarController` based app)

If we set the height of custom bar by setting `preferredContentSize` to anything more than ~100-120, then it starts to push the `UITabBar` downwards (clipped from the bottom) and there is a slight gap visible between the `UITabBar` of the and the `PopUpBar`. You may have a look at the attached screenshot where I reproduced it in the example app (size = (-1, 300)). 

The gap increases as we increase the height of the `PopUpBar`.

<a href="https://ibb.co/geCoP5"><img src="https://preview.ibb.co/h44PxQ/IMG_0588.png" alt="IMG_0588" border="0"></a>

Steps to reproduce:
1. Create a subclass of `LNPopUpCustomBarViewController` for example `DemoPopUpBarViewController`.

2. Write down the following code in this controller:
`    self.preferredContentSize = CGSizeMake(-1, 300);
`
3. Write down this code at the bottom of `_presentBar` method of `FirstViewController` of the example project.
```
    DemoPopUpBarViewController* barVc = [[DemoPopUpBarViewController alloc] initWithNibName:@"DemoPopUpBarViewController" bundle:nil];
    targetVC.popupBar.customBarViewController = barVc;
    [targetVC presentPopupBarWithContentViewController:demoVC animated:YES completion:nil];
```

Device: iPhone 6 Plus
iOS version: 10.3.1

I hope it gets fixed sooner.

Thanks,
Atul

**Comment by LeoNatan** [comment]
Looks like similar to #167.

I will investigate soon,

**Comment by LeoNatan** [comment]
I’m not sure I will have to investigate this for the 2.5.0 milestone, as I want to finish the iPhone X + iOS 11 support and another minor fix. This will get priority after that.

**Comment by LeoNatan** [comment]
It’s actually a small fix and will be included in 2.5.0.

**Comment by LeoNatan** [comment]
Fixed. Please test 2.5.0 release and let me know.
